includes/migration/wc/class-fs-wc-migration.php

```diff
@@ -237,12 +212,7 @@ protected function get_local_install_id() {
 			 * Limit the ID to 32 chars since the entity mapping
 			 * local_id column is limited to 32 chars.
 			 */
-			return substr(
-				$this->_edd_download->ID . '_' .
-				md5( $this->get_edd_canonized_site_home_url() ),
-				0,
-				32
-			);
+			return $this->ep->site_uid;
```

Comment:
@shramee `site_uid` and `install_id` are NOT the same things. Please generate the `install_id` based on a similar structure I created the EDD `insatll_id`.